auditorium/src/views/components/auditorium/privacy.js

```diff
@@ -22,27 +22,29 @@ const Privacy = (props) => {
   if (userHasOptedIn) {
     deleteButton = (
       <Fragment>
-        <p
-          class='ma0 mb3'
-          dangerouslySetInnerHTML={{
-            __html: __('Stay opted in, only delete <strong>usage data</strong>')
-          }}
-        />
-        <button
-          class='pointer w-100 w-auto-ns f5 link dim bn dib br1 ph3 pv2 mr1 mb4 white bg-mid-gray'
-          data-role='purge'
-          onclick={handlePurge}
-        >
-          {__('Delete')}
-        </button>
+        <div class='bl-ns b--moon-gray pl4-ns'>
```

Comment:
Now that this is wrapped in a `div` anyways, the wrapping `Fragment` is not needed anymore.